### ISSUE TITLE
Display Waschmaschine Remaining Program Time in hours and minutes

### ISSUE THREAD

**Issue by FlavorFx** [DavidMStraub/homeassistant-homeconnect#35]
Display the remaining time in hours and minutes, not seconds.

**Comment by DavidMStraub** [comment]
I agree that it's annoying to have remaining times of thousands of seconds. However, I believe this is a frontend issue. This can easily be solved in a custom card (which would anyway make sense to have at some point) or by using a template sensor. IMO it would even be nice if HA by default formatted sensors with time values in a friendly manner.

Fixing this at the component level would mean turning the numerical sensor into a string sensor, and I don't think this would be a good idea. It would e.g. prevent automations with a numeric state trigger.

Using minutes as numerical unit instead of second also wouldn't help since for coffee machines, the remaining program time is always just a few seconds.

To summarize, I think it's an issue of presentation that we cannot solve at the level of the component.

**Comment by DavidMStraub** [comment]
This is solved in 0.3 by using a timestamp sensor.
